devito/operator.py

```diff
@@ -188,6 +192,36 @@ def __init__(self, expressions, **kwargs):
 
     # Compilation
 
+    def _add_implicit(self, expressions):
+        """
+        Create and add to the expression tree any associated implicit
```

Comment:
nitpicking: it's not an "expression tree" yet, it's just an orderer sequence of expressions.